modules/learn/pages/clusters-and-availability/xdcr-conflict-resolution.adoc

```diff
@@ -1,49 +1,68 @@
+
+NOTE: XDCR optimistic replication behavior applies only when the `enableCrossClusterVersioning` property is not in use.
+
+Once configured, conflict resolution is a fully automated process, which requires no manual intervention.
+
+[#eccv-true-for-conflict-resolution-process]
+=== When enableCrossClusterVersioning is true for a Conflict Resolution Process
+
+XDCR optimistic replication behavior, as described earlier, applies only when the 'enableCrossClusterVersioning' property is not in use. 
+
+When the `enableCrossClusterVersioning` property is set to `true` (enabled), the hybrid logical vector (HLV) metadata is created and maintained for each document in the document xattrs (system created extended attributes). This HLV metadata needs to be checked and updated for both the source and the target documents. As a result, the XDCR processing retrieves metadata from the target cluster for every document of any size that undergoes replication. For more information on the `enableCrossClusterVersioning` property and the HLV metadata, see xref:clusters-and-availability/xdcr-enable-crossclusterversioning.adoc[XDCR enableCrossClusterVersioning].
 
 [#revision-id-based-conflict-resolution]
 == Conflict Resolution Based on Sequence Number
 
 Conflicts can be resolved by referring to documents' _sequence numbers_.
 Sequence numbers are maintained per document, and are incremented on every document-update.
-A document's sequence number is stored as part of its _metadata_: specifically, as the value of the `rev` key (see xref:manage:manage-ui/manage-ui.adoc#console-documents[Documents], for details on how to inspect metadata).
+A document's sequence number is stored as a part of its _metadata_: specifically, as the value of the `rev` key (see xref:manage:manage-ui/manage-ui.adoc#console-documents[Documents], for details on how to inspect metadata).
 The sequence numbers of source and target documents are compared; and the document with the higher sequence number prevails.
 If both documents have the same sequence number, the conflict is resolved by comparing the following metadata-elements, in the order shown:
 
 . CAS value
 . Expiration (TTL) value
 . Document flags
 
+[#eccv-true-in-sequence-number-based-conflict-resolution]
+=== When enableCrossClusterVersioning is true for a sequence number based conflict resolution process 
+
+When the `enableCrossClusterVersioning` property is set to `true`, the hybrid logical vector (HLV) metadata, located in the document xattrs (system created extended attributes) of the source and the target, is also used in the sequence number based conflict resolution processing. The features or options used in the XDCR process determine the specific ways of using the HLV metadata. For more information on the `enableCrossClusterVersioning` property and the HLV metadata, see xref:clusters-and-availability/xdcr-enable-crossclusterversioning.adoc[XDCR enableCrossClusterVersioning].
```

Comment:
Remove lines 61-64.
Add a line that says:
When Cross Cluster Versioning is enabled, the hybrid logical vector (HLV) metadata in the source and target document's xattr, is also used in the conflict resolution processing. For more information on the enableCrossClusterVersioning property and the HLV metadata, see xref:clusters-and-availability/xdcr-enable-crossclusterversioning.adoc[XDCR enableCrossClusterVersioning].